Source/WebKit/UIProcess/RemoteLayerTree/mac/RemoteScrollingTreeMac.mm

```diff
@@ -123,6 +123,11 @@
 void RemoteScrollingTreeMac::didCommitTree()
 {
     ASSERT(isMainRunLoop());
+    ASSERT(m_treeLock.isLocked());
```

Comment:
This could probably be `WTF_REQUIRES_LOCK(m_treeLock)` in the header for a compile-time guarantee instead of runtime.